Source/WebCore/rendering/svg/RenderSVGContainer.cpp

```diff
@@ -133,8 +133,7 @@ void RenderSVGContainer::paint(PaintInfo& paintInfo, const LayoutPoint& paintOff
 
     auto adjustedPaintOffset = paintOffset + currentSVGLayoutLocation();
     if (paintInfo.phase == PaintPhase::Mask) {
-        // FIXME: [LBSE] Upstream SVGRenderSupport changes
-        // SVGRenderSupport::paintSVGMask(*this, paintInfo, adjustedPaintOffset);
+        SVGRenderSupport::paintSVGMask(*this, paintInfo, adjustedPaintOffset);
```

Comment:
I think it's time to move them into a shared place, that is not SVGRenderSupport, but REnderLayerModelObject.h. There is also a section in that header, which already contains LBSE specific functions shared across RenderSVG* renderers. Let's move paintSVGClipMask and paintSVGMask there :-)

Reply:
You are right, that way we can also make it non-static. Done.